package.json

```diff
@@ -39,6 +39,8 @@
     "g:check-trace": "cd $INIT_CWD && yarn run -T rimraf tsconfig.tsbuildinfo && yarn run -T tsc -p tsconfig.json --generateTrace traceDir && yarn run -T analyze-trace traceDir"
   },
   "devDependencies": {
+    "@babel/core": "^7.18.2",
+    "@babel/generator": "^7.18.2",
```

Comment:
I think it would be better to use it for packages that actually have dependencies, like `packages/babel`.